### ISSUE TITLE
[FEATURE] Expose Cadence 1.0 CLI Install/Usage docs

### ISSUE THREAD

**Issue by jribbink** [onflow/docs#660]
### Issue to be solved

Either add a page for this in our dev docs or link to https://cadence-lang.org/docs/cadence_migration_guide.

**Comment by SupunS** [comment]
Yeah, it would be nice to have the Cadence 1.0 CLI Installation steps in a separate page/section, so we can link it from other places. For e.g: it is currently repeated in multiple places:
- https://github.com/onflow/cadence-lang.org/blob/bf3a09b70dd23bf1c977e9e2f4044310f538fe81/docs/cadence_migration_guide/index.md?plain=1#L19-L38
- https://github.com/onflow/cadence-lang.org/blob/08437c7f924bffac3d845e7ea08b9984aa7d2547/docs/cadence_migration_guide/emulator-state-migration-guide.mdx#L42
